src/styles/index.scss

```diff
@@ -0,0 +1,8 @@
+@import "./blocks/card";
+@import "./blocks/stars";
+@import "./utils/reset";
+@import "./blocks/header";
+@import "./blocks/menu";
+@import "./utils/grid";
+
+
```

Comment:
when adding imports it's important to add firstly all the utils, for example variables and reset and then all blocks in which you use those styles